DEPLOYING.md

```diff
@@ -31,8 +31,8 @@ signing.keyId=<8-character-public-key-id>
 signing.password=<key-password>
 signing.secretKeyRingFile=<your-home-directory>/.gnupg/secring.gpg
 
-ossrhUsername=<ossrh-username>
-ossrhPassword=<ossrh-password>
+grpc.deploy.ossrh.username=<ossrh-username>
```

Comment:
I was thinking this one wouldn't be gRPC specific. If you have multiple projects, you likely want this to be shared across them.

Since ossrhUsername is what OSSRH [documents](http://central.sonatype.org/pages/gradle.html#credentials), it seems best to use it instead of making our own.